README.md

```diff
@@ -1 +1,45 @@
-# charmed-superset-rock
\ No newline at end of file
+# Charmed Apache Superset ROCK
+
+This repository contains the packaging metadata for creating a Charmed Superset ROCK. This ROCK image is based on the upstream [Apache Superset](https://downloads.apache.org/superset/) image.
+
+For more information on ROCKs, visit the [rockcraft Github](https://github.com/canonical/rockcraft).
+
+## Building the ROCK
+The steps outlined below are based on the assumption that you are building the ROCK with the latest LTS of Ubuntu.  
+If you are using another version of Ubuntu or another operating system, the process may be different. 
+To avoid any issue with other operating systems you can simply build the image with [multipass](https://multipass.run/):
+```bash
+sudo snap install multipass
+multipass launch 22.04 -n rock-dev
```

Comment:
`22.04` will not be the latest LTS forever but I guess it is not that important.

Reply:
When we get a new LTS release I'll need to test the rock on this version and then can update the `README`